lib/y2_module_guitest.pm

```diff
@@ -48,8 +48,9 @@ sub launch_yast2_module_x11 {
         script_run('pkill -TERM -e yast2');
         select_console('x11');
     }
+    my $yast_env_variables = y2_module_basetest->with_yast_env_variables();
```

Comment:
Doesn't it work with `$self->with_yast_env_variables()` ?

Reply:
I guess it should even work with just plain call `with_yast_env_variables()`, as the function is exported in `y2_module_basetest.pm`.

Reply:
> I guess it should even work with just plain call `with_yast_env_variables()`, as the function is exported in `y2_module_basetest.pm`.

Even better :)

Reply:
@mloviska no, it doesnt and i am not sure why http://aquarius.suse.cz/tests/1480

Reply:
@b10n1k, because
>Global symbol "$self" requires explicit package name (did you forget to declare "my $self"?) at /var/lib/openqa/share/tests/sle/lib/y2_module_consoletest.pm line 13.

You did not declare `$self` variable (e.g. with `my $self = shift;`)

Reply:
and also with just the name fails http://aquarius.suse.cz/tests/1482#step/yast2_clone_system/5